paddle/fluid/operators/conv_op.cc

```diff
@@ -187,7 +187,10 @@ framework::OpKernelType ConvOp::GetExpectedKernelType(
     customized_type_value =
         (input_data_type == framework::DataTypeTrait<int8_t>::DataType() ||
          input_data_type == framework::DataTypeTrait<uint8_t>::DataType())
-            ? kConvMKLDNNINT8
+            ? OperatorWithKernel::IndicateVarDataType(ctx, "Filter") ==
+                      framework::DataTypeTrait<int8_t>::DataType()
+                  ? kConvMKLDNNINT8WS8
+                  : kConvMKLDNNINT8
```

Comment:
Does `kConvMKLDNNINT8` need to be changed to `kConvMKLDNNINT8WU8`?

Reply:
kConvMKLDNNINT8 specifies the regular conv as worked previously - it is int8 conv but the weights are initially in fp32  and converted in first iter. 
kConvMKLDNNINT8WS8 was added to differentiate a convolution after the pass which has weights stored in int8_t (and biases in int32_t). 
Therefore, if I were to change the name  kConvMKLDNNINT8 it would have to be kConvMKLDNNINT8WFP32 or kConvMKLDNNINT8WF32.
But then I should probably also touch other operators for consistency if I begin to change names. If we're going to add more operators in future maybe that's not a bad idea.